### ISSUE TITLE
Curator 4.1.2 can install elasticsearch-py 5.0.0

### ISSUE THREAD

**Issue by PhaedrusTheGreek** [elastic/curator#798]
Running Curator 4.1.2 Against ES 2.3 yields the following error (`Requirement.parse('elasticsearch<5.1.0,>=2.4.0')`), even though [docs](https://github.com/elastic/curator#compatibility-matrix) suggest otherwise. 

```
Traceback (most recent call last):
  File "c:\users\kramases\appdata\local\programs\python\python35\lib\site-packages\pkg_resources\__init__.py", line 635,
in _build_master
    ws.require(__requires__)
  File "c:\users\kramases\appdata\local\programs\python\python35\lib\site-packages\pkg_resources\__init__.py", line 943,
in require
    needed = self.resolve(parse_requirements(requirements))
  File "c:\users\kramases\appdata\local\programs\python\python35\lib\site-packages\pkg_resources\__init__.py", line 834,
in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (elasticsearch 2.3.0 (c:\users\kramases\appdata\local\programs\python\python35\
lib\site-packages), Requirement.parse('elasticsearch<5.1.0,>=2.4.0'), {'elasticsearch-curator'})
```

**Comment by untergeek** [comment]
That's because of a python version mismatch. I'm guessing someone installed
via pip and it installed v5.0.0 of the Elasticsearch python module.

This is already fixed in master. It can be fixed by running

pip install -U elasticsearch==2.4.0

-- Aaron Mildenstein

**Comment by untergeek** [comment]
Duplicate of #787, fixed in #788.
